### PR TITLE
Add scripts to automate setting up IIB images for testing

### PR DESCRIPTION
**Pull request by amisevsk**
### What does this PR do?
Add scripts:
* `installCatalogSourceFromIIB.sh` : given an IIB image, install it in an OpenShift Cluster as a catalogsource and optionally install the DevSpaces Operator
* `testLatestIIB.sh` : automatically get the latest IIB for a given version of DevSpaces, and install a DevSpaces cluster in the current OpenShift cluster.

Usage examples:
1. Install CatalogSource for a specific IIB image in the `devspaces-testing` namespace, and install the `devspaces` operator from it
    ```bash
    ./installCatalogSourceFromIIB.sh \
      --iib registry-proxy.engineering.redhat.com/rh-osbs/iib:225370 \
      -n devspaces-testing \
      --install-operator devspaces \
      --channel stable
    ```
    This script can be used to test any IIB image (e.g. DevWorkspace Operator)

2. Install DevSpaces cluster from the latest DS 3.0 IIB image in namespace `devspaces-testing` and print dashboard URL once install succeeds:
    ```bash
    ./testLatestIIB.sh -t 3.0 -n devspaces-testing --get-url
    # ...eventually outputs
    DevSpaces is installed
    Dashboard URL...........https://devspaces-devspaces-testing.apps.<snip>.rhcloud.com
    Devfile registry URL....https://devspaces-devspaces-testing.apps.<snip>.rhcloud.com/devfile-registry
    Plugin registry URL.....https://devspaces-devspaces-testing.apps.<snip>.rhcloud.com/plugin-registry/v3
    Workspaces base domain..apps.<snip>.rhcloud.com
    ```

### What issues does this PR fix or reference?
I don't think we created one

### CODE REVIEW

File: product/testLatestIIB.sh [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+if [[ "$CREATE_CHECLUSTER" == "false" ]]; then
+  echo "Not creating CheCluster -- all done"
+  exit 0
+fi
+
+echo "Waiting for CheCluster CRs to be available in the cluster. Timeout is 3 minutes."
+for _ in {1..60}; do
+  echo -n '.'
+  oc get crd checlusters.org.eclipse.che > /dev/null 2>&1 && break
+  sleep 3
```
Comment (nickboldt) [inline product/testLatestIIB.sh:101]: this is 3 seconds.

File: product/testLatestIIB.sh [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+done
+echo ""
+
+if ! oc get crd checlusters.org.eclipse.che > /dev/null 2>&1; then
+  errorf "DevSpaces operator install not completed within 3 minutes; giving up"
```
Comment (nickboldt) [inline product/testLatestIIB.sh:106]: please `s/DevSpaces/Dev Spaces/g` so the Branding Gods don't rain fire down on us.

File: product/installCatalogSourceFromIIB.sh [hunk 3]
```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
```
Comment (nickboldt) [inline product/installCatalogSourceFromIIB.sh:3]: pfft. copyright dates. these don't need to be accurate statements of creation & maintenance. :P

Comment (amisevsk) [reply product/installCatalogSourceFromIIB.sh:3]: Hey, they're copy-pasted from other files in the repo :stuck_out_tongue:

**Pull request merged**